### DISCUSSION TITLE
ImportError: cannot import name 'deprecated' from 'typing_extensions'

### DISCUSSION THREAD

**Discussion by vladmandic** [vladmandic/sdnext#1610] [General]
If you receive above error during initial install/startup of SD.Next, simply run `webui.bat/webui.sh` again and installer will cleanup dependencies. We're still investigating where incorrect dependency gets first installed, but good thing is that second start cleans up any errors.

**Comment by lbeltrame** [comment]
It looks like some kind of race to me, because the second run does not download / install anything but it goes through. I assume SD.Next does some `sys.path` trickery? (I didn't look at the code yet).

**Comment by vladmandic** [comment]
Yes it does, but this issue just started popping up without any relevant backend change, so still need to figure out why.

My guess is that some other package has a file called exactly like that so it's old is it a file or a module python problem.
